include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.cpp

```diff
@@ -175,6 +175,59 @@ TimeIntBDF<dim, Number>::advance_one_timestep_partitioned_solve(bool const use_e
   Base::advance_one_timestep_solve();
 }
 
+template<int dim, typename Number>
+void
+TimeIntBDF<dim, Number>::get_quantities_and_times(
+  std::vector<VectorType const *> &                             quantities,
+  std::vector<double> &                                         times,
+  std::function<VectorType const *(unsigned int const)> const & get_quantity) const
+{
+  /*
+   *
+   *   time t
+   *  -------->   t_{n-2}   t_{n-1}   t_{n}     t_{n+1}
+   *  _______________|_________|________|___________|___________\
+   *                 |         |        |           |           /
+   *               sol[2]    sol[1]   sol[0]
```

Comment:
you could think about renaming `sol` -> `quantities`.

Reply:
[54e19d5](https://github.com/exadg/exadg/pull/458/commits/54e19d55e77b5520afe808311d28e9403135757f)